ansible/roles/docker/README.md

```diff
@@ -0,0 +1,40 @@
+docker
+=========
+
+A role for installing Docker.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+None
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+
+Here's how to use it in a playbook:
+
+    - hosts: mongo
```

Comment:
Did you mean to put "mongo" in here or is that a copy/paste error?

Reply:
Good catch, @dav3r! :trophy: